docs/backends/bind.rst

```diff
@@ -2,13 +2,19 @@ BIND zone file backend
 ======================
 
 * Native: Yes
-* Master: Yes
-* Slave: Yes
-* Superslave: Experimental
+* Primary: Yes
+* Secondary: Yes
+* Producer: Yes
+* Consumer: Yes
+* Autoprimary: Yes
```

Comment:
every backend that can Primary ("send NOTIFY, serve AXFR") can Autoprimary. Listing it separately is not necessary.

Reply:
I added that because the [overview table](https://doc.powerdns.com/authoritative/backends/index.html) has such a column. In that table there are some backends with `Primary: Yes` but `Autoprimary: No`. These are wrong then? And dropping Autoprimary from that table would be sensible then, right?